resource/gauges_2.ui

```diff
@@ -29,4 +29,4 @@
  </widget>
  <resources/>
  <connections/>
-</ui>
+</ui>
```

Comment:
Small unintended change

Reply:
So there is no change to be done from what I understand.

Reply:
Check the tab where you review the pr

Reply:
Oh I see.